_posts/doc/2014-02-01-raspberry-pi-quick-start.md

```diff
@@ -97,6 +97,7 @@ following steps:
     <pre>admin</pre>
 
 ## Development Environment Installation
+### User Workspace
```

Comment:
You should add an empty line before this, in order to have it correctly rendered.